dotnet/notebooks/config/MODELS.md

```diff
@@ -0,0 +1,27 @@
+# Dotnet Notebooks - Supported Models
+The tables below show which models are available and supported with the dotnet notebooks. Please select from these tables when configuring your service variables to run the notebooks.
```

Comment:
I'm wondering whether we want to maintain this table ourselves our just point to the documentation from OpenAI and Azure OpenAI?

https://platform.openai.com/docs/models/overview
https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models